drupal/acquia/update_hp_paragraphs.sh

```diff
@@ -0,0 +1,7 @@
+#updates all of the Howard Packages (https://packagist.org/packages/howard/)
+composer update 'howard/howard_paragraphs'
```

Comment:
Is there a way to use a wildcard when updating packages?

Reply:
I'm not sure. Why would you need a wildcard to update the packages?

Reply:
Yes, you could, see [docs](https://getcomposer.org/doc/03-cli.md#update-u).

Reply:
also, the key here is (according to https://bugify.idfive.com/issues/22972), that this be done on "all Acquia d8 environments". So ideally, we would have the script update both hud8 and academic departments. So something like:
- `cd hud8-dev`
- `composer update 'howard/*'`
- `cd ../academicdepartments-dev`
- - `composer update 'howard/*'`

The idea being, update all howard pagages, everywhere. Otherwise we could simply run `composer update 'howard/*'` in each codebase, and the script wouldnt be all that useful. Perhaps even add a git commit to this as well, ie, in each folder:
- `git add -a`
- `git commit -m "Updating all howard packages via HAL script"`

Reply:
Actually, in the interest of time, I went ahead and wrote all this out, so we could move it forward on the other scripts. See https://github.com/howard-university-web-services/Howard-Automation-Library/blob/master/drupal/acquia/update_howard_packages.sh for how all the above changes have been implemented for this feature (update all Howard packagist modules). Feel free to close this PR once reviewed, and we can use the above as a starting point for the next script.